src/functions/advanced-functions/organizer.php

```diff
@@ -13,8 +13,11 @@
 if ( class_exists( 'Tribe__Events__Main' ) ) {
 
 	/**
-	 * Create an Organizer
+	 * Create an Organizer using the legacy method.
 	 *
+	 * Note: This function is outdated and should be replaced with the [TEC ORM `tribe_organizers()->create()` method](https://docs.theeventscalendar.com/apis/orm/create/organizers/).
```

Comment:
Should we be adding deprecations here...?

Reply:
@stratease  the deprecation tag is currently removing the function completely from our docs so until that is fixed @bordoni says we should add a note to the DocBlock to at least give people a heads up that it will be deprecated soon